testdata/p4tc_samples_outputs/direct_counter_example.json

```diff
@@ -0,0 +1,91 @@
+{
+  "schema_version" : "1.0.0",
+  "pipeline_name" : "direct_counter_example",
+  "externs" : [
+    {
+      "name" : "DirectCounter",
+      "id" : 1,
```

Comment:
Here also

```suggestion
      "id" : 101,
```

Reply:
I have hardcoded it for now but going further we need to decide the extern id according to specs.